xml/System/MathF.xml

```diff
@@ -989,10 +1400,30 @@ The value of this field is 3.14159274.
         <Parameter Name="x" Type="System.Single" />
       </Parameters>
       <Docs>
-        <param name="x">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="x">A single-precision floating-point number to be rounded.</param>
+        <summary>Rounds a single-precision floating-point value to the nearest integral value, and rounds midpoint values to the nearest even number.</summary>
+        <returns>The integer nearest <paramref name="x" />. If the fractional component of <paramref name="x" /> is halfway between two integers, one of which is even and the other odd, then the even number is returned. Note that this method returns a <see cref="T:System.Single" /> instead of an integral type.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+
+This method uses the default rounding convention of <xref:System.MidpointRounding.ToEven?displayProperty=nameWithType>. See [Midpoint values and rounding conventions](#midpoint-values-and-rounding-conventions) for information about rounding numbers with midpoint values.
```

Comment:
```suggestion
This method uses the default rounding convention of <xref:System.MidpointRounding.ToEven?displayProperty=nameWithType>.
```
The **Midpoiint rounding** section is in the overloaded method portion of the Math.Round documentation and not in this topic.